PipelinesTasks/PRMetrics/buildAndReleaseTask/invokers/azureReposInvoker.ts

```diff
@@ -0,0 +1,219 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import { IGitApi } from "azure-devops-node-api/gitApi";
+import { JsonPatchDocument, JsonPatchOperation, Operation } from "azure-devops-node-api/interfaces/common/VSSInterfaces";
+import { Comment, CommentThreadStatus, GitPullRequest, GitPullRequestCommentThread, GitPullRequestIteration } from "azure-devops-node-api/interfaces/GitInterfaces";
+import DevOpsApiWrapper from "../wrappers/devOpsApiWrapper";
+import TaskLibWrapper from "../wrappers/taskLibWrapper";
+
+export default class AzureReposInvoker {
```

Comment:
I copied a different example that did the export at the end. Out of curiosity, why did you go with this instead?

Reply:
Exporting at the end might be useful it you encapsulate the class with something but in our case it has no difference.

Reply:
Makes sense, I'm making the change to the rest of our classes now for consistency